README.md

```diff
@@ -76,6 +70,8 @@ username:ega-test-data@ebi.ac.uk
 password:egarocks
 ```
 
+`Note` : If no credential file were provided it will prompt the user for username and password.
```

Comment:
Shouldn't it be "If no credential file was provided" or "If no credential files were provided"? Sorry for nitpicking.